src/main/java/com/example/schedule/demo/mapper/ScheduleMapper.java

```diff
@@ -13,4 +15,7 @@ public interface ScheduleMapper {
     @Select("SELECT * FROM schedules")
     public List<Schedule> findAll();
 
+    @Insert("insert into schedules(title,scheduleDate,scheduleTime) Values (#{title},#{scheduleDate},#{scheduleTime})")
```

Comment:
すごい細かいのですが気がついてしまったので...
```suggestion
    @Insert("insert into schedules(title,scheduleDate,scheduleTime) values (#{title},#{scheduleDate},#{scheduleTime})")
```

Reply:
修正しました。
https://github.com/aokiayukodesu/Schedule-list/pull/2/commits/be87f3c3eceb43a8b9f09fe51a9cf2f6d4ae0c16